src/Traversals/Traversals.jl

```diff
@@ -21,42 +21,36 @@ show(io::IO, ::TraversalError) = println(io, "An error was encountered while tra
 
 """
     abstract type TraversalAlgorithm
-
```

Comment:
can you revert these changes? these blank lines in the docstrings are deliberate.